src/components/ResultsItem/ResultsItem.css

```diff
@@ -48,4 +48,19 @@
 
 .results-item-footer {
     font-size: 12px;
-}
\ No newline at end of file
+}
+
+#parent{position: relative;}
```

Comment:
this isn't too important, but I see some inconsistencies with how your code is formatted (not just in CSS but just in general) compared to the rest of our files, like spacing inconsistencies for example (missing spaces, extra newlines). this won't impact how the code runs so it's fine, but something that would be nice to do if possible is to use an automatic code formatter like Prettier, which you could install as a vscode extension if you use vscode!